src/app/features/persons/person-dialog/person-profiles/person-profiles-musician/person-profiles-musician.component.ts

```diff
@@ -56,33 +59,32 @@ export class PersonProfilesMusicianComponent implements OnInit {
     if (this.person) {
       this.tableData.next(this.person.musicianProfiles);
     }
-    // ToDo: Mira 17.11. Eine solche Select Value Category gibt es nicht. Kann also nicht funktionieren.
-    this.instrumentOptions$ = this.selectValueService
-      .load('Sections', 'Name')
-      .pipe(map(() => this.selectValueService.get('Sections', 'Name')));
-  }
+   }
 
-  // onSubmit() {
-  //   if (this.person) {
-  //     const { instrumentId } = this.form.getRawValue();
-  //       this.muproService
-  //         .addMusicianProfile(this.person.id, { instrumentId })
-  //         .pipe(first())
-  //         .subscribe((result) => {
-  //           if (this.person && !this.person?.musicianProfiles) {
-  //             this.person.musicianProfiles = [];
-  //           }
-  //           this.person?.musicianProfiles?.push(result);
-  //           this.tableData.next(this.person?.musicianProfiles);
-  //           this.notificationsService.success('MUSICIAN_PROFILE_ADDED', 'person.mupro');
-  //           this.form.reset({});
-  //         });
-  //     }
-  //   }
 
   createNewMuPro() {
-    this.router
-      .navigate(['/arpa', 'mupro', this.person?.id, { outlets: { modal: ['create', this.person?.id] } }])
-      .then(() => this.ref.close(false) );
+    const ref = this.dialogService.open(MusicianInstrumentsComponent, {
+      data: {
+        profile: new BehaviorSubject({doublingInstruments: [],
```

Comment:
Hier wäre es gut der Konvention zu folgen und BehaviorSubjects mit $$-Präfix zu versehen, also `profile$$`. Achtung, muss dann auch in src/app/features/musician-profile-dialog/musician-instruments/musician-instruments.component.ts angepasst werden.